packages/react-dom/package.json

```diff
@@ -30,6 +30,7 @@
     "README.md",
     "build-info.json",
     "index.js",
+    "testing.js",
```

Comment:
Putting this here means it will go into the next public release. If you can't get rid of it because the bundle tests rely on it, maybe prefix with `unstable`?